### PR TITLE
handling window frame spec for window query

### PR DESCRIPTION
**Pull request by suryasoma**

### CODE REVIEW

File: spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/TestConstants.java [hunk 1]
```diff
@@ -491,4 +491,15 @@ private static <T> T[] copy(T... elements) {
               3L, 5L, 6L, ImmutableList.of(new StringStruct("6:str6", "5:str5", "3:str3"))),
           new NumStruct(
               3L, 1L, 4L, ImmutableList.of(new StringStruct("3:str3", "1:str1", "4:str4"))));
+
+  public static final List<NumStruct> numStructDatasetForUnion =
```
Comment (VinayLondhe14) [inline spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/TestConstants.java:495]: Shouldn't this be `numStructDatasetForWindow`?

**Review by VinayLondhe14 — APPROVED** [review]

File: spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/TestConstants.java (hunk 1 above)
Comment (suryasoma) [reply spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/TestConstants.java:495]: my bad. will change it

**Pull request merged**